src/index.js

```diff
@@ -6,6 +6,29 @@ export default function register(Component, tagName, propNames, options) {
 		inst._vdomComponent = Component;
 		inst._root =
 			options && options.shadow ? inst.attachShadow({ mode: 'open' }) : inst;
+
+		if (options && options.shadow && options.injectGlobalStyles) {
+			const defaults = {
+				target: document.head,
+				selector:
+					'style, link[rel="stylesheet"], link[rel="preload"][as="style"]',
```

Comment:
Strings tend to not compress well. If we switch to `document.styleSheets` and match our selectors on `sheet.ownerNode`, we should be able to get rid of it.